instrumentation/camel-2.20/javaagent/src/test/groovy/io/opentelemetry/javaagent/instrumentation/apachecamel/aws/CamelSpan.groovy

```diff
@@ -23,22 +23,6 @@ class CamelSpan {
     }
   }
 
-  static sqsProduce(TraceAssert traceAssert, int index, queueName, parentSpan = null) {
-    return traceAssert.span(index) {
-      name queueName
-      kind INTERNAL
-      if (index == 0) {
-        hasNoParent()
-      } else {
-        childOf parentSpan
-      }
-      attributes {
-        "camel.uri" "aws-sqs://${queueName}?amazonSQSClient=%23sqsClient&delay=1000"
-        "$SemanticAttributes.MESSAGING_DESTINATION_NAME" queueName
-      }
-    }
-  }
-
   static sqsConsume(TraceAssert traceAssert, int index, queueName, parentSpan = null) {
```

Comment:
Is this still used then?

Reply:
yes this is used by both the [S3CamelTest](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/camel-2.20/javaagent/src/test/groovy/io/opentelemetry/javaagent/instrumentation/apachecamel/aws/S3CamelTest.groovy#L46) and the [SnsCamelTest](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/camel-2.20/javaagent/src/test/groovy/io/opentelemetry/javaagent/instrumentation/apachecamel/aws/SnsCamelTest.groovy#L38). Do you have an opinion on what should be done with those ignored tests?

Reply:
Can we check history and see how long they've been ignored? If it's been a while I think it's fine to remove them (in a separate PR).

Reply:
As explained in the comment on these tests they are ignored because they don't work with localstack. Deleting them does not seem like a good idea.

Reply:
> As explained in the comment on these tests they are ignored because they don't work with localstack. Deleting them does not seem like a good idea.

Are they being run tho?

Reply:
I can look into converting them to java along with everything else, even if we leave them ignored

Reply:
They are not run, you'd need to run these tests against actual aws infra which we don't have. It certainly is less than ideal to have tests that don't run but nevertheless these tests would be invaluable to the person who needs to debug the features that these tests cover.

Reply:
all tests have now been converted